src/Symfony/Bundle/TwigBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * Inject `twig` service on `AppVariable` by calling `setTwig`
```

Comment:
Not sure that's something we need to document as it's mainly an implementation detail, isn't it?